src/App.java

```diff
@@ -1,31 +1,270 @@
+            String person = input.nextLine();
+            System.out.print(BLUE + "│ " + RESET + "Enter a place: \n" + BLUE + "│ ↪ " + RESET);
+            String place2 = input.nextLine();
+            System.out.print(BLUE + "│ " + RESET + "Enter a verb ending in -ing: \n" + BLUE + "│ ↪ " + RESET);
+            String verbIng2 = input.nextLine();
+            System.out.print(BLUE + "│ " + RESET + "Enter an object: \n" + BLUE + "│ ↪ " + RESET);
+            String object3 = input.nextLine();
+            System.out.print(BLUE + "│ " + RESET + "Enter an object: \n" + BLUE + "│ ↪ " + RESET);
+            String object4 = input.nextLine();
+            System.out.print(BLUE + "│ " + RESET + "Enter a person: \n" + BLUE + "│ ↪ " + RESET);
+            String person2 = input.nextLine();
+            System.out.print(BLUE + "│ " + RESET + "Enter a verb: \n" + BLUE + "│ ↪ " + RESET);
+            String verb = input.nextLine();
+            System.out.print(BLUE + "│ " + RESET + "Enter an adjective: \n" + BLUE + "│ ↪ " + RESET);
+            String adj2 = input.nextLine();
+
+            System.out.println(BLUE + "├────────────────────────────────────────" + RESET);
+            System.out.println(BLUE + "│ " + RESET + "Yesterday, " + celeb + " went to the " + place + " and started "
+                    + verbIng + ". \n" +
+                    BLUE + "│ " + RESET + "However, " + hero
+                    + " was there and was able to stop him before it was too late.\n" +
+                    BLUE + "│ " + RESET + "He grabbed a " + object + " and used it to apprehend " + celeb + ".\n" +
+                    BLUE + "│ " + RESET + "But, " + celeb2 + " appeared out of nowhere with a " + adj + " " + object2
+                    + " and helped " + celeb
+                    + " escape!\n" +
+                    BLUE + "│ " + RESET + "\n" +
+                    BLUE + "│ " + RESET + "The next day, " + person + " was walking towards the " + place2
+                    + " when they saw " + celeb
+                    + " and " + celeb2 + ".\n" +
+                    BLUE + "│ " + RESET + "They were each holding a " + object3 + " and " + object4 + " and were "
+                    + verbIng2 + " " + person2
+                    + "!\n" +
+                    BLUE + "│ " + RESET + "It was a " + adj2 + " sight to see, but " + person
+                    + " knew they had to do something, so he called over " + hero + ". \n" +
+                    BLUE + "│ " + RESET + "Together, they were able to " + verb + " " + celeb + " and " + celeb2
+                    + " and save " + person2
+                    + "! The end.\n" +
+                    BLUE + "│ " + RESET);
+
+            System.out.print(BLUE + "│ " + GREEN + "Do you want to play again? (y/n)\n" + BLUE + "│ ↪ " + RESET);
+            String playAgain = input.nextLine();
+            if (playAgain.equals("y")) {
+                doMadlib();
```

Comment:
this is a **recursive call**, which is a little bit risky - it's not a great idea, especially in large functions like this, for a method to call itself like this unless it's a small, clearly organized function (and even then plenty of people would say recursion is mostly bad, even though we will learn more about it later). A loop would be a safer choice. That said, it's a pretty clever solution for an early project.